build/automation/var/platform-texas/v1/account-live-k8s-prod.mk

```diff
@@ -1,5 +1,8 @@
 AWS_ACCOUNT_ID := $(AWS_ACCOUNT_ID_PROD)
 AWS_ACCOUNT_NAME := prod
+AWS_CERTIFICATE = 8d65eee4-cf92-4a00-84de-7a9f544ba724
+
+WAF_ACL_ID = ff530a4e-689d-4d90-b3ab-ac2160b5863d
```

Comment:
Is this the Texas (prod) WAF? If so, that's the right place for this config.

Reply:
Done